solr/core/src/java/org/apache/solr/cli/AssertTool.java

```diff
@@ -123,6 +123,14 @@ public List<Option> getOptions() {
         Option.builder("e")
             .desc("Return an exit code instead of printing error message on assert fail.")
             .longOpt("exitcode")
+            .build(),
+        // u was taken, can we change that instead?
+        Option.builder("credentials")
```

Comment:
excuse the possibly dumb question. why is SolrCLI.OPTION_CREDENTIALS not available to use here?

Reply:
Because the `u` short option is already used elsewhere in the assert tool....

Reply:
ah yeah, and I missed your comment too 👍